ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -682,5 +682,12 @@ final class Metrics(val registry: MetricRegistry) {
         val prune: Timer = registry.timer(Prefix :+ "prune")
       }
     }
+
+    object http_json_api {
```

Comment:
Please don't use underscore in class or object names.
```suggestion
    object httpJsonApi {
```

Reply:
Adressed with 169de2eb1b310000636dac82e4e4e72da476044c